### ISSUE TITLE
New Octane Rule: `no-classic-components`

### ISSUE THREAD

**Issue by patocallaghan** [ember-cli/eslint-plugin-ember#542]
Pulled from https://github.com/ember-cli/eslint-plugin-ember/issues/533 which proposes a new set of rules for Octane related best practices

Disallow usage of the `@ember/component` base class. Use `@glimmer/component` instead.

```js
// ### Good
import Component from '@glimmer/component'

// ### Bad
import Component from '@ember/component'
```

If anyone would like to implement this please comment that you'll take it and work away.

**Comment by patocallaghan** [comment]
@rwjblue @Turbo87 checking the import location should be enough here?

**Comment by rwjblue** [comment]
> checking the import location should be enough here?

Yeah, I think that is the best we can do.

**Comment by scalvert** [comment]
Agree. I think the import is pretty conclusive here.

**Comment by mikoscz** [comment]
I'm taking this one :)

**Comment by patocallaghan** [comment]
@bmish could you assign this issue to @mikoscz please?
